ui/src/app/cohort-review/detail-tabs/detail-tabs.component.ts

```diff
@@ -139,156 +156,173 @@ export class DetailTabsComponent implements OnChanges, OnInit, OnDestroy {
-      sourceVocabulary: sourceVocabulary,
-      visitId: visitId,
+      visitType: visitType,
     }
   }, {
-    name: 'Measurements',
-    domain: DomainType.MEASUREMENT,
+    name: 'Observations',
+    domain: DomainType.OBSERVATION,
     filterType: PageFilterType.ReviewFilter,
     columns: [
-      itemDate, standardName, standardCode, standardVocabulary, valueConcept, valueNumber,
-      valueSource, units, ageAtEvent, labRefRange, sourceName, sourceCode, visitId
+      itemDate, standardName, standardCode, standardVocabulary, ageAtEvent, sourceName,
+      sourceCode, sourceVocabulary, visitId
     ],
     reverseEnum: {
       itemDate: itemDate,
       standardName: standardName,
       standardCode: standardCode,
       standardVocabulary: standardVocabulary,
-      valueConcept: valueConcept,
-      valueNumber: valueNumber,
-      valueSource: valueSource,
-      units: units,
       age: ageAtEvent,
-      labRefRange: labRefRange,
       sourceName: sourceName,
       sourceCode: sourceCode,
+      sourceVocabulary: sourceVocabulary,
       visitId: visitId,
     }
   }, {
-    name: 'Observations',
-    domain: DomainType.OBSERVATION,
+    name: 'Physical Measurements',
+    domain: DomainType.PHYSICALMEASURE,
     filterType: PageFilterType.ReviewFilter,
     columns: [
-      itemDate, standardName, standardCode, standardVocabulary, ageAtEvent, sourceName,
-      sourceCode, sourceVocabulary, visitId
+      itemDate, standardCode, standardVocabulary, standardName, value, unit, ageAtEvent
     ],
     reverseEnum: {
       itemDate: itemDate,
-      standardName: standardName,
       standardCode: standardCode,
       standardVocabulary: standardVocabulary,
+      standardName: standardName,
+      value: value,
+      unit: unit,
       age: ageAtEvent,
-      sourceName: sourceName,
-      sourceCode: sourceCode,
-      sourceVocabulary: sourceVocabulary,
-      visitId: visitId,
     }
   }, {
-    name: 'Physical Measurements',
-    domain: DomainType.PHYSICALMEASURE,
+    name: 'Lab',
+    domain: DomainType.LAB,
     filterType: PageFilterType.ReviewFilter,
     columns: [
-      itemDate, standardName, standardCode, standardVocabulary, valueConcept, valueNumber,
-      valueSource, units, ageAtEvent
+      itemDate, itemTime, standardName, value, unit, refRange, ageAtEvent, visitType
     ],
     reverseEnum: {
       itemDate: itemDate,
+      itemTime: itemTime,
       standardName: standardName,
-      standardCode: standardCode,
-      standardVocabulary: standardVocabulary,
-      valueConcept: valueConcept,
-      valueNumber: valueNumber,
-      valueSource: valueSource,
-      units: units,
+      value: value,
+      unit: unit,
+      refRange: refRange,
+      age: ageAtEvent,
+      visitType: visitType
+    }
+  }, {
+    name: 'Vital',
+    domain: DomainType.VITAL,
+    filterType: PageFilterType.ReviewFilter,
+    columns: [
+      itemDate, standardName, value, unit, refRange, ageAtEvent, visitType
```

Comment:
Should `itemTime` be included in the vitals columns?

Reply:
Done.